index.html

```diff
@@ -305,8 +330,8 @@ <h1 class="myself-header">
         <h3>Contact me</h3>
```

Comment:
- [ ] This should use a header tag of `h2` tags instead of `h1`. Think of these header tags as Title > subtitle > .....

Ensure the hierarchy is maintained.